src/main/java/ome/services/fulltext/FullTextBridge.java

```diff
@@ -330,6 +362,26 @@ public void set_folders(final String name, final IObject object,
         }
     }
 
+    /**
+     * Walks the {@link Fileset} instances attached to and Image
+     * so users can use the original clientpath for search.
+     */
+    public void set_fileset(final String name, final IObject object,
+                            final Document document, final LuceneOptions opts) {
+        if (object instanceof Image) {
+            final Image image = (Image) object;
+            final Fileset fileset = image.getFileset();
+            final Iterator<FilesetEntry> entryIterator = fileset.iterateUsedFiles();
```

Comment:
This line threw NPE in CI.

Reply:
Cheers. Did you happen to notice if that made the indexer particularly unhappy?

Reply:
It did. Unfortunately, in this case restarting it wouldn't help as the exception's repeatable. (At least it logged the exception!)

At the moment truly unexpected things stop it in its tracks. It's an unpleasant tradeoff between that and possibly regarding a bunch of failed things as having been indexed; a difficult design issue.

Reply:
That's what I assumed after running into exceptions while testing this. Happy to just be as safe as possible before merging this, but if we could find somewhere fairly close to `FullTextBridge.set` (perhaps in that method itself) for capturing all NPEs and similar it might be worth it.

Reply:
For indexing failures better to WARN and actually move on? Sounds plausible; I can do that if desired.

Reply:
(Actually, even ERROR could be good for those.)

Reply:
It's an interestingly difficult one: if it instead throws `UnresolvableObjectException` then retrying will probably work. Have to give it some thought.